connectors/src/connectors/zendesk/temporal/activities.ts

```diff
@@ -36,6 +38,8 @@ export async function saveZendeskConnectorStartSync(connectorId: ModelId) {
   if (res.isErr()) {
     throw res.error;
   }
+
+  return { isInitialSync: !connector.lastSyncSuccessfulTime };
```

Comment:
Open question: should we maybe base this on `zendesk_timestamp_cursors` rather than `connectors`?
In this case we could get `isInitialSync` outside of this activity and right before launching the incremental sync, which would be simpler IMHO

Reply:
(I changed it to what you suggested btw :p)